test/framework/fixtures/libgoalFixture.go

```diff
@@ -324,6 +333,27 @@ func (f *LibGoalFixture) ShutdownImpl(preserveData bool) {
 	}
 }
 
+// dumpLogs prints out node.log files for the running nodes
+func (f *LibGoalFixture) dumpLogs(filePath string) {
+	file, err := os.Open(filePath)
+	if err != nil {
+		f.t.Logf("could not open %s node.log", filePath)
+		return
+	}
+	var lines []string
+	scanner := bufio.NewScanner(file)
+	for scanner.Scan() {
+		lines = append(lines, scanner.Text())
```

Comment:
I think you can count lines in the loop and break earlier

Reply:
How? We want the last 100 lines of the log, not the first 100 correct?

Reply:
or, right, my bad.